### PR TITLE
New presentation for inflected form search results

### LINKED ISSUE #399: Systematize presentation of additional info for (inflected) word form and dictionary entry
Currently, we show the inflectional category (Verb, Noun, Particle as the general classes, and VTA-1-5 etc. as the specific subcategory) in conjunction with the inflected form, when that information comes from the dictionary source and actually concerns the dictionary entry head. For the inflectional subcategory information presented in conjunction with the dictionary head it would be useful to have an `(i)` button to show additional information.

On the other hand, the `(i)` button with the morphological/inflectional breakdown is shown in the appropriate place next to the inflected wordform, and should be kept there.

So, we should modify and extend the current presentation to something like the following (note that we are showing the word-form with morphological boundaries, which has its own separate issue):

image

In the above, 1) _ni·kî-nitawi-wâpam·âw_ is the inflected word-form, for which the 2) linguistic inflectional breakdown applies (as is currently the case). In contrast, 3) _wâpamêw_ is the dictionary entry head, for which the 4a) general (`Verb`) and 4b) specific inflectional subcategory (`like: wîcihêw`, using the plain English label) information applies. In the underlying database, 3a is specified by labels, in this case `VTA-1`. However, `crk.altlabel.tsv` contains alternative relabelings, of which by default we would want to use the plain English one (here: `like: wîcihêw`). However, we could allow for providing additional information, say the code (`VTA-1`) as well as the long linguistic label (`transitive animate verb – class 1: regular`) by using a `(i)` button, e.g. showing these together as `transitive animate verb – class 1: regular (VTA-1)` or `VTA-1: transitive animate verb – class 1: regular`

As a glimpse of further development later on, through settings or otherwise, one might want to be able to choose also the full linguistic (`transitive animate verb – class 1: regular`) or plain nêhiyawêwin (`tâpiskôc: wîcihêw`) labels as the primary ones shown.

### PR DESCRIPTION
**Pull request by eddieantonio**
Fixes #399 

 - [x] shift word class down to "elaboration"
 - [x] add Cree explanation for word class
 - [x] ensure the word class is still visible for lemmas

This is what it looks like...

~~for the lemma form of verbs~~: 
Screen Shot 2020-06-05 at 1 36 16 PM

**EDIT**: this is what it looks like for verbs now (consistent with non-lemma forms):
Screen Shot 2020-06-05 at 3 13 09 PM

for inflected forms of verbs:
Screen Shot 2020-06-05 at 1 36 28 PM

for nouns:
Screen Shot 2020-06-05 at 1 37 40 PM

for particles/preverbs:
Screen Shot 2020-06-05 at 1 35 59 PM

### CODE REVIEW

**Comment by aarppe** [comment]
Two questions, which can be addressed now, or later:
1. For the inflectional subcategory, are we using Arok's inflectional classes that should be in our internal dictionary database, i.e. VII/VAI/VTI/VTA-[1-5], and their relabelings in `crk.altlabel.tsv`? (Rather than the specific wordclass (that comes from the linguistic analysis, and which also has relabelings in `crk.altlabel.tsv`)
2. Sh/Couldn't we provide the inflectional subcategory for nouns as well? Their inflectional subcategory codes should be available, and the relabelings for them should exist in `crk.altlabel.tsv`.

**Comment by eddieantonio** [comment]
> Two questions, which can be addressed now, or later:
> 
> 1. For the inflectional subcategory, are we using Arok's inflectional classes that should be in our internal dictionary database, i.e. VII/VAI/VTI/VTA-[1-5], and their relabelings in `crk.altlabel.tsv`? (Rather than the specific wordclass (that comes from the linguistic analysis, and which also has relabelings in `crk.altlabel.tsv`)
> 2. Sh/Couldn't we provide the inflectional subcategory for nouns as well? Their inflectional subcategory codes should be available, and the relabelings for them should exist in `crk.altlabel.tsv`.
> 
> Also note that the relabeling `like: wîcihêw, itêw` is rather the plain English label for the specific word class, than the Cree one - this Cree labeling would rather be: `tâpiskôc: wîcihêw, itêw`. But I wouldn't use these specific wordclasses, but rather the inflectional subcategories, since they tell more about how the word inflectes, and the relabelings are actually shorter - see for VTAs:
> 
> ```
> VTA-1	VTA-1	transitive animate verb – class 1: regular	like: wîcihêw	tâpiskôc: wîcihêw
> VTA-2	VTA-2	transitive animate verb – class 2	like: nitonawêw	tâpiskôc: nitonawêw
> VTA-3	VTA-3	transitive animate verb – class 3	like: kîskiswêw	tâpiskôc: kîskiswêw
> VTA-4	VTA-4	transitive animate verb – class 4	like: nakatêw	tâpiskôc: nakatêw
> VTA-5	VTA-5	transitive animate verb – class 5	like: itêw	tâpiskôc: itêw
> ```
> 
> As far as I understand, this information is already available, since it is used to disambiguate paradigm generation.

Antti: by "inflectional subcategory", did you mean [inflectional **category**](https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/blob/master/docs/glossary.md#inflectional-category)? Or are you referring to something else?

**Comment by aarppe** [comment]
Yes, drop the _sub-_, so it should be inflectional category, i.e. `NA-1`, `VTA-1`, `NI-4w`.

**Comment by eddieantonio** [comment]
> Yes, drop the _sub-_, so it should be inflectional category, i.e. `NA-1`, `VTA-1`, `NI-4w`. Corrected that in the original comment as well.

Thank you for clarifying!

File: cypress/integration/search.spec.js [hunk 1]
```diff
@@ -369,4 +369,73 @@ context('Searching', () => {
         .should('contain', NON_WORD)
     })
   })
+
+  describe('display of the header', function () {
```
Comment (Madoshakalaka) [inline cypress/integration/search.spec.js:373]: amazing test

**Comment by eddieantonio** [comment]
> LGTM. Just a little busy-looking with such a long line "Verb ---- like: blah blah"

Yeah, for sure! Antti's re-labellings for each particular inflectional class are shorter, so hopefully that will make things less busy in #434

File: cypress/integration/search.spec.js (hunk 1 above)
Comment (eddieantonio) [reply cypress/integration/search.spec.js:373]: names are hard, mkay?

**Pull request merged**